component-models.json

```diff
@@ -1530,7 +1533,9 @@
         "name": "numberOfArticles",
         "label": "Number Of Articles Display",
         "valueType": "number",
-        "value": 0
+        "validation": {
+          "numberMin": 0
```

Comment:
@mayur-satav Can we change the min number to 3 or more? The 0 will not display the results.

Reply:
we had call and decided to keep min 1